pkg/api/migrate_machine.go

```diff
@@ -2,41 +2,70 @@ package api
 
 import (
 	"encoding/json"
+	"log"
 	"net/http"
+
+	"github.com/leapp-to/leapp-go/pkg/executor"
 )
 
+// MigrateParams represents the parameters sent by the client
 type MigrateParams struct {
-	SourceHost       string            `json:"source_host,omitempty"`
-	TargetHost       string            `json:"target_host,omitempty"`
-	ContainerName    string            `json:"container_name,omitempty"`
-	SourceUser       string            `json:"source_user,omitempty"`
-	TargetUser       string            `json:"target_user,omitempty"`
-	ExcludePaths     []string          `json:"excluded_paths,omitempty"`
-	TcpPorts         map[uint16]uint16 `json:"tcp_ports,omitempty"`
-	ExcludedTcpPorts []uint16          `json:"excluded_tcp_ports,omitempty"`
-	ForceCreate      bool              `json:"force_create,omitempty"`
-	DisableStart     bool              `json:"disable_start,omitempty"`
-	Debug            bool              `json:"debug,omitempty"`
+	StartContainer    bool     `json:"start_container"`
+	ContainerName     string   `json:"container_name"`
+	ForceCreate       bool     `json:"force_create"`
+	SourceHost        string   `json:"source_host"`
+	SourceUser        string   `json:"source_user"`
+	TargetHost        string   `json:"target_host"`
+	TargetUser        string   `json:"target_user"`
+	ExcludePaths      []string `json:"excluded_paths"`
+	UseDefaultPortMap bool     `json:"use_default_port_map"`
```

Comment:
Why did you remove the Excluded tcp ports and the TcpPorts? This needs to be kept here and passed to the executor in buildActorInput